src/components/Dashboard/widgets/Activity/ActivityTable.tsx

```diff
@@ -17,12 +17,13 @@ const ActivityTitle = styled(WidgetTitle)`
 `;
 
 const ActivityTable = (): ReactElement => {
-  const utxos = useSelector(selectUnspentUtxos);
+  let txs = useSelector(selectParsedTransactions);
+  txs = txs.length > 3 ? txs.slice(0, 3) : txs;
```

Comment:
You don't need use a ternary here, `Array.slice` won't error out if you give it an array shorter than the second arg.